flocx_market/.env

```diff
@@ -0,0 +1,2 @@
+DATABASE_URI=mysql+pymysql://flocx_market:qwerty123@127.0.0.1:3307/flocx_market
+FLASK_APP=api.py
```

Comment:
These files would be replaced by the oslo_config stuff that @ljmcgann  and @DanNiESh  are working on.

Reply:
got it